src/gantry_calibration/calibrate.py

```diff
@@ -0,0 +1,227 @@
+"""
+Gantry Calibration Algorithm Demo.
+
+Usage:
+    calibrate.py basic
+    calibrate.py bilinear [<points>]
+    calibrate.py biquadratic [<points>]
+    calibrate.py bicubic [<points>]
+    calibrate.py poly <x_order> <y_order> <x_points> <y_points>
+    calibrate.py -h | --help
+
+Options:
+    -h --help           Show help screen.
+
+For more information see README.md
+"""
+
+import sys
+from typing import Callable
+from collections import namedtuple
+import numpy as np
+import matplotlib.pyplot as plt  # type: ignore
+from docopt import docopt
+from calibration import Point, PointPair, Calibration
+
+Travel = namedtuple("Travel", ["min", "max"])  # Range of travel of each axis
+
+TRAVEL_MIN = 0.0
+TRAVEL_MAX = 1.0
+TRAVEL_RANGE = TRAVEL_MAX - TRAVEL_MIN
+ERROR_FRACTION = 0.05
```

Comment:
Comments would be nice for these globals. I think you can also delete TRAVEL_RANGE since it should be calculated internally.

Reply:
Done and done.